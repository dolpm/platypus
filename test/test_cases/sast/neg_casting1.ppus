pipe main [] |> unit {
  str x = Str_new <| ["I LOVE TUFTS <3 <3"];
  float y = Float <| [x];

  |> ();
}
