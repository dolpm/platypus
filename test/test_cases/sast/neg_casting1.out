Fatal error: exception Failure("unexpected arg type str in Float")
